classes/controller/FrontController.php

```diff
@@ -2198,4 +2228,9 @@ protected function validateInputAsUrl($data): bool
 
         return Validate::isUrl($data);
     }
+
+    protected function getControllerName(): string
```

Comment:
Hi @Kaikina, I love it :) 
Can you please move this fonction to the abstract ControllerCore please ?
And I'm not sure that using php_self is appropriated here

Reply:
php_self is the best identifier normally, but I think we could just use the controller classname/filename instead?